client/stylesheets/_toCat.scss

```diff
@@ -4347,14 +4347,14 @@ On click the images will be displayed at normal size to complete the effect
 
 
 #profileReceivedOffersMaps {
-    margin-top: 20px;
+    margin-top: 10px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered height, margin-top, width